### ISSUE TITLE
Shake not working

### ISSUE THREAD

**Issue by grumphus** [marcowindt/WiiMoteDSU#14]
Hello. Thank you so much for developing this awesome tool. Perhaps the only thing that I can't seem to get working on my Samsung Galaxy S9+ is the shake.

I don't think it's a hardware limitation because MotionSource detects it perfectly and the DSU Controller Test is a lot smoother compared to WiiMoteDSU.

I tried fiddling with the different options to no avail. Are there plans for any improvements in this area? Thank you!

**Comment by silver-volt4** [comment]
Yes, the app's got pretty bad sensitivity.
What I ended up doing was using a holy marriage of these two apps - I made MotionSource act as a server on a different port, and then pointed dolphin to use MotionSource's server for the motion, and WiiMoteDSU's server for the wiimote buttons. (I ran both apps at the same time, MotionSource can run in the background when it's a server.) It works pretty well, I got great results in Wii Sports Resort.

It's pretty tedious to set it up, getting Dolphin to recognize the servers being the hardest part.
